.circleci/config.yml

```diff
@@ -52,9 +52,11 @@ workflows:
       - ensure_version_bump:
           requires:
             - build
+          filters:
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
+            branches:
+              ignore: /.*/
```

Comment:
@cpjolicoeur does it make sense to only bump version numbers when we're releasing tags for corresponding npm releases? That way master branch doesn't always have to fail when we're not bumping the version.